js_modules/dagster-ui/packages/ui-core/src/runs/QueuedRunCriteriaDialog.tsx

```diff
@@ -0,0 +1,164 @@
+import {
+  Box,
+  Button,
+  Colors,
+  Dialog,
+  DialogBody,
+  DialogFooter,
+  Icon,
+  Mono,
+  Table,
+  Tag,
+  Tooltip,
+} from '@dagster-io/ui-components';
+import * as React from 'react';
+import {Link} from 'react-router-dom';
+import * as yaml from 'yaml';
+
+import {RunTableRunFragment} from './types/RunTable.types';
+import {useRunQueueConfig} from '../instance/useRunQueueConfig';
+import {StructuredContentTable} from '../metadata/MetadataEntry';
+
+type TagConcurrencyLimit = {
+  key: string;
+  value?: string;
+  limit: number;
+};
+
+export const QueuedRunCriteriaDialog = ({
+  isOpen,
+  onClose,
+  run,
+}: {
+  isOpen: boolean;
+  onClose: () => void;
+  run: RunTableRunFragment;
+}) => {
+  const runQueueConfig = useRunQueueConfig();
+  const runTagMap = React.useMemo(() => {
+    const tagMap: {[key: string]: string} = {};
+    run.tags.forEach((tag) => {
+      tagMap[tag.key] = tag.value;
+    });
+    return tagMap;
+  }, [run.tags]);
+  const maxConcurrentRuns = runQueueConfig?.maxConcurrentRuns;
+  const runTagLimits = React.useMemo(() => {
+    try {
+      if (!runQueueConfig?.tagConcurrencyLimitsYaml) {
+        return undefined;
+      }
+      const limits: [TagConcurrencyLimit] = yaml.parse(runQueueConfig.tagConcurrencyLimitsYaml);
+      return limits.filter(
+        (limit) =>
+          limit.key in runTagMap &&
+          (limit.value === undefined || limit.value === runTagMap[limit.key]),
+      );
+    } catch (err) {
+      return undefined;
+    }
+  }, [runQueueConfig, runTagMap]);
+  const priority = runTagMap['dagster/priority'];
+  const runIsOpConcurrencyLimited =
+    runQueueConfig?.isOpConcurrencyAware &&
+    run.rootConcurrencyKeys &&
+    run.rootConcurrencyKeys.length > 0 &&
+    !run.hasUnconstrainedRootNodes;
+
+  return (
+    <Dialog
+      isOpen={isOpen}
+      title="Run queue criteria"
+      canOutsideClickClose
+      canEscapeKeyClose
+      onClose={onClose}
+    >
+      <DialogBody>
```

Comment:
Let the table be full-bleed by just using a `div` here, then you don't need to add an extra right border?